main.js

```diff
@@ -2,3 +2,28 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+const characterList = $('.item');
+
+function getCharacters() {
+    for (let i = 0; i <= 4; i++) {
```

Comment:
be careful using loops here. this is made so that the API call happens 4 time every time. this will waste unnecessary resources to get the same information over and over.

you should make this loop inside the .then(res => {